### ISSUE TITLE
Feature request: support sixel also without big libraries

### ISSUE THREAD

**Issue by Sytoplis** [ranger/ranger#3030]
Hello, I have been starting to use ranger today and wanted previews for images. I am currently running on Wayland and many options were not available. I decided to use Sixel, as my teminal supports that, but I was sad to find that it is only possible by using the ImageMagick library, which is very big and i wanted to use something more light weight. I then tried to get it working with the way smaller library libsixel-utils, which provides the command 'img2sixel' to do basically everything that is necessary. So I've played around in the ext/img_display.py file to try to get it to work and it indeed did, except that the aspect ratio is not preserved and I sadly didn't find a way to fix it. If anyone has an idea, i would be glad to further investigate.

TL;DR: I got mostly the same functionality with a smaller library. Maybe you can add support for this library as well.

```python
"""
#--- BEGIN MAGICK COMMAND --- (what i have replaced)
            environ.setdefault("MAGICK_OCL_DEVICE", "true")
            try:
                check_call(
                    [
                        *MAGICK_CONVERT_CMD_BASE,
                        path + "[0]",
                        "-geometry",
                        "{0}x{1}>".format(fit_width, fit_height),
                        "-dither",
                        sixel_dithering,
                        "sixel:-",
                    ],
                    stdout=cached,
                    stderr=DEVNULL,
                    env=environ,
                )
            except CalledProcessError:
                raise ImageDisplayError("ImageMagick failed processing the SIXEL image")
            except FileNotFoundError:
                raise ImageDisplayError("SIXEL image previews require ImageMagick")
            finally:
                cached.flush()

            if os.fstat(cached.fileno()).st_size == 0:
                raise ImageDisplayError("ImageMagick produced an empty SIXEL image file")
            #--- END MAGICK COMMAND ---
"""
            #WHY THIS IS WORSE THAN USING MAGICK: the reason i ended up using magick is that magic actually can resize the images to keep the aspect ratio. This is AFAIK not possible with img2sixel and the small library given, although the command below DOES in fact produce previews.

            #NOTE: this is my custom version to not use ImageMagick, but instead use only libsixel and libsixel-utils with the command 'img2sixel'

            
            #--- MY OWN VERSION USING img2sixel ---
            try:
                check_call(
                    [
                        "img2sixel",
                        path,
                        "--width={0}".format(fit_width),
                        "--height={0}".format(fit_height)
                    ],#TODO: add dithering form the settings by using the --diffusion option
                    stdout=cached,
                    stderr=DEVNULL,
                    env=environ,
                )
            except CalledProcessError:
                raise ImageDisplayError("img2sixel failed processing the SIXEL image")
            except FileNotFoundError:
                raise ImageDisplayError("SIXEL image previews require img2sixel from libsixel-utils")
            finally:
                cached.flush()

            if os.fstat(cached.fileno()).st_size == 0:
                raise ImageDisplayError("img2sixel produced an empty SIXEL image file")
            #--- END OF MY OWN BODGED VERSION ---
```

The stuff i changed was here: https://github.com/ranger/ranger/blob/bd9b37faee9748798f3970468bd0d1dedbf0e99f/ranger/ext/img_display.py#L478

PS: I also tried to put it into the commands.py file, but for some reason it didnt recognize it there.
PPS: This is mainly for other people who also would like to use a smaller library. Maybe it is useful to someone.
